### PR TITLE
Origin/unit test/greeting controller

### PR DESCRIPTION
**Pull request by egemahlec**
Please review my modifications and let me know your suggestions. I wrote unit test for GreetingController class and I generated gradle wrapper so that every developer work with same gradle version. From now on, there is no need to install gradle. Please just call ./gradlew instead of gradle command. I will update readme.md file later.

### CODE REVIEW

File: API Project/src/main/resources/config/application.properties [hunk 1]
```diff
@@ -1,11 +0,0 @@
-# https://spring.io/guides/gs/accessing-data-mysql/
```
Comment (yigitozkavci) [inline API Project/src/main/resources/config/application.properties]: This file is required for standalone mysql client access, why did you remove this?

File: API Project/build.gradle [hunk 2]
```diff
@@ -21,15 +21,31 @@ repositories {
...
+
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
-    // testCompile('org.springframework.boot:spring-boot-starter-test')
+    testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile("org.hsqldb:hsqldb")
```
Comment (yigitozkavci) [inline API Project/build.gradle]: I don't think we should add hsqldb as dependency since we use mysql, should we?

**Review by yigitozkavci — CHANGES_REQUESTED** [review]

File: API Project/src/main/resources/config/application.properties (hunk 1 above)
Comment (egemahlec) [reply API Project/src/main/resources/config/application.properties]: While merging the branches, I accidentally deleted this file. Thank you for your comment.
